app/api/upload/PDF.js

```diff
@@ -28,6 +28,21 @@ export default class PDF extends EventEmitter {
     };
   }
 
+  async createThumbnail(documentId) {
+    let response;
+    try {
+      response = await spawn(
+        'pdftoppm',
+        ['-f', '1', '-singlefile', '-scale-to', '320', '-jpeg', this.filepath, path.join(path.dirname(this.filepath), documentId)],
+        { capture: ['stdout', 'stderr'] }
+      );
+    } catch (err) {
+      debugLog.debug(`Thumbnail creation error for: ${this.filepath}`);
```

Comment:
This probably needs to go into the errorLog instead?

Reply:
i think this should throw the error and let the route deal with it, it is catched there and there is a process already implemented for errors on PDF conversion [here](https://github.com/huridocs/uwazi/blob/1903-PDF-Thumbnails/app/api/upload/routes.js#L80) ?  or do we want to handle this in a diferent way because its only the thumbnail  ?